### ISSUE TITLE
.NET 6 RC1: Option<string[]> no longer associated with multiple consecutive arguments

### ISSUE THREAD

**Issue by guitarrapc** [dotnet/format#1391]
## tl;dr;

`dotnet format --exclude foo bar srn.sln` could not handle `foo` and `bar` as --exclude's inputs.

## Summary

Looking into https://github.com/dotnet/command-line-api/issues/1352 , dotnet/format already specifies `AllowMultipleArgumentsPerToken = true` for array options `--include`, `--exclude`, and `--diagnostics`. 
However could not handle  multiple values on .NET 6 RC1 SDK.

> https://github.com/dotnet/format/blob/e492bc50564242ddde6e4aab2d8e69cc6c0cd82f/src/Commands/FormatCommandCommon.cs#L51

## Step to reproduce

I've created sample repo to reproduce issue.

```shell
$ git clone https://github.com/guitarrapc/dotnet-format-issue-report
$ cd dotnet-format-issue-report
$ dotnet format --exclude foo bar src.sln
```

## Actual result

`bar` is not recognized as input for `--exclude`.

```shell
$ dotnet --version
6.0.100-rc.1.21458.32

$ dotnet format --exclude foo bar src.sln
Unrecognized command or argument 'srn.sln'
Description:
  Formats code to match editorconfig settings.

Usage:
  dotnet [options] format [<PROJECT | SOLUTION>] [command]

Arguments:
  <PROJECT | SOLUTION>  The project or solution file to operate on. If a file is not specified, the command will search 
                        the current directory for one. [default: C:\dotnet\src\]

.... omitted ...
```

## Expected result

both `foo` and `bar` is not recognized as input for `--exclude`.

Issue not happen with nuget's .NET SDK 5 version, so you can confirm expected result with this.

```shell
$ dotnet tool install -g dotnet-format
$ dotnet-format --version
5.1.250801+4a851ea9707f87d381166c2fc2b2d4b58a10a222
$ dotnet-format --exclude foo bar srn.sln
  Formatting code files in workspace 'C:\dotnet\src\src.sln'.
  Format complete in 2755ms.
```

## Workaround

* If `--exclude` input is single parameter, nothing trouble. `dotnet format --exclude foo srn.sln`.
* If 2 or more parameter is required, use dotnet global tool's doenet-format instead.

# Environment

* OS: Windows 10
* dotnet SDK; 6.0.100-rc.1.21458.32

**Comment by guitarrapc** [comment]
Confirmed issue fixed in latest .NET 6 RC1 SDK.

```
$ dotnet --version
6.0.100-rc.2.21505.57
$ dotnet format --exclude foo bar src.sln
$ 
```
